src/sink/udp.rs

```diff
@@ -78,7 +85,16 @@ impl Sink for Udp {
                 let raw = codec.encode(value)?;
                 for processed in postprocess(&mut self.postprocessors, ingest_ns, raw)? {
                     //TODO: Error handling
-                    socket.send(&processed).await?;
+                    if self.config.bound {
+                        socket.send(&processed).await?;
+                    } else {
+                        socket
+                            .send_to(
+                                &processed,
+                                (self.config.dst_host.as_str(), self.config.dst_port),
```

Comment:
would it be faster/more efficient to store a ready SocketAddr in the config instead of having to go through the `into()` dance for this tuple for every package?

Reply:
Ah, i think the into dance includes the address resolving here, so this actually makes sense at this point. Could you add a comment on that?